ibis/backends/bigquery/__init__.py

```diff
@@ -755,6 +755,7 @@ def insert(
             Name of the attached database that the table is located in.
         overwrite
             If `True` then replace existing contents of table
+
```

Comment:
Is this intentional?

Reply:
It's from ruff format.

Reply:
So, no I didn't personally intend to do this, on the other hand I don't feel like fighting ruff about it.

Reply:
I guessed so, I think it is dependent on the ruff version.